senator/__main__.py

```diff
@@ -1,8 +1,62 @@
 from senator.utility import *
 from senator.game import *
+import argparse
+import numpy as np
 
-votes, initial_utility = parse_data(load_data(FILE_PATH))
 
-game = Game(initial_utility,votes)
+def main() -> tuple[np.ndarray, np.ndarray, np.ndarray]:
 
-game.run_greedy_search()
\ No newline at end of file
+    parser = argparse.ArgumentParser(
+        description="Run the senator game simulation with specified parameters.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--num_steps",
+        type=int,
+        default=7,
+        help="Number of steps for which the greedy algorithm is run.",
+    )
+
+    parser.add_argument(
+        "--owner_loss",
+        type=float,
+        default=0.9,
+        help="Loss for the owner.",
+    )
+    parser.add_argument(
+        "--other_loss", type=float, default=0.8, help="Loss for others."
+    )
+    parser.add_argument(
+        "--owner_trashold",
+        type=float,
+        default=-1,
+        help="Threshold for the owner. Set to -1 to change the model.",
```

Comment:
Change the model to what? A butterfly? A unicorn? Another model?

Reply:
Unicorn would be nice!

Fair point, will change that.